src/NuGet.Core/NuGet.Build.Tasks.Pack.Library/PackTask.cs

```diff
@@ -79,9 +84,12 @@ public override bool Execute()
         {
             var request = GetRequest();
             var logic = PackTaskLogic;
-
+            PackageBuilder packageBuilder = null;
             var packArgs = logic.GetPackArgs(request);
-            var packageBuilder = logic.GetPackageBuilder(request);
+            if (string.IsNullOrEmpty(request.NuspecFile))
```

Comment:
yes, as i said in the description, i'll update the PR with tests once https://github.com/NuGet/NuGet.Client/pull/1116 is checked in